aws_ssh_tunnel.py

```diff
@@ -109,12 +109,20 @@ def set_target_instance_details(ctx, session):
         sys.exit(1)
 
 
-def get_free_port():
+def get_free_port(requested_port):
```

Comment:
The name get_free_port implies that it returns a randomly available port. I would consider changing the name of this method to something like: get_port

Reply:
A good suggestion; I did so just now.